go/ct/utils/adapter_test.go

```diff
@@ -313,9 +313,9 @@ func TestAdapter_ParameterConversion(t *testing.T) {
 
 	for name, property := range properties {
 		t.Run(name, func(t *testing.T) {
-			state := st.State{}
-			property.set(&state)
-			params := ToVmParameters(&state)
+			state := st.NewState(st.NewCode([]byte{}))
+			property.set(state)
+			params := ToVmParameters(state)
```

Comment:
This change looks unrelated ... why is it needed?

Reply:
because just calling `st.State{}` does not allocate for underlying structures, such as `Storage`, `Accounts` or `BlockContext`, I noticed `set` functions do this, so I'll revert this change and add the necessary calls to `NewBlockContext`